autoload/neomake/makers/ft/swift.vim

```diff
@@ -1,7 +1,24 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#swift#EnabledMakers() abort
-    return ['swiftc']
+    if filereadable('Package.swift')
+        return ['swiftpm']
+    else
+        return ['swiftc']
+    endif
```

Comment:
Should use `neomake#utils#FindGlobFile` - see usages with other makers.

Reply:
Updated!